modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/cmake-lists.mustache

```diff
@@ -27,28 +25,55 @@ endif()
 # BUILD TYPE
 message("A ${CMAKE_BUILD_TYPE} build configuration is detected")
 
-# Update require components as necessary
-#find_package(Boost 1.45.0 REQUIRED COMPONENTS ${Boost_THREAD_LIBRARY} ${Boost_SYSTEM_LIBRARY} ${Boost_REGEX_LIBRARY} ${Boost_DATE_TIME_LIBRARY} ${Boost_PROGRAM_OPTIONS_LIBRARY} ${Boost_FILESYSTEM_LIBRARY})
+if (UNIX)
```

Comment:
I would indent all statements inside if clauses by 2 or 4 spaces, so it's obvious it's if-else